userspace/falco/webserver.cpp

```diff
@@ -68,6 +65,13 @@ void falco_webserver::start(
             res.set_content(versions_json_str, "application/json");
         });
 
+    if (metrics.is_enabled())
+    {
+        m_server->Get("/metrics",
+            [metrics](const httplib::Request &, httplib::Response &res) {
+                res.set_content(metrics.to_text(), "text/plain; version=0.0.4");
```

Comment:
What is the `0.0.4` version? If it's some constant out of our control, can we define it as a constant somewhere?

Reply:
It's the [text based exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format).
Agreed, I'll make it a documented constant.